src/group/group.service.ts

```diff
@@ -96,7 +97,13 @@ export class GroupService {
       hasNext,
     };
 
-    return { bookmarks, pagination };
+    return {
+      bookmarks: bookmarks.map((bookmark) => ({
+        ...bookmark,
+        tags: bookmark.tags.map((tag) => tag.tag),
```

Comment:
Destructuring `tag` to `({ tag })` might looks better.

Reply:
Good Idea! 👏🏻